.github/workflows/cont_integration.yml

```diff
@@ -62,8 +61,44 @@ jobs:
         cargo update -p tokio --precise "1.38.1"
     - name: Build
       run: cargo build --features ${{ matrix.features }} --no-default-features
-    - name: Clippy
-      if: ${{ matrix.rust.clippy }}
-      run: cargo clippy --all-targets --features ${{ matrix.features }} --no-default-features -- -D warnings
     - name: Test
       run: cargo test --features ${{ matrix.features }} --no-default-features -- --test-threads=1
+
+  fmt:
+    name: Rust Formatting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Rust Toolchain
+        uses: dtolnay/rust-toolchain@v1
+        with:
+            toolchain: stable
+            components: rustfmt
+      - name: Check fmt
+        run: cargo fmt --all -- --config format_code_in_doc_comments=true,wrap_comments=true --check
```

Comment:
It might be good to have a `rustfmt.toml` for fine-tuning the config, for example I think wrapping comments to a default width of 80 columns is a little short and generally prefer 90 or 100.

Reply:
I like the idea of having `fmt` do the `wrap_comments` even if they're a bit shorter than we'd like. It will be one less thing for comment writers to have to think about and make sure is uniform.

Reply:
Sweet! I added the `.rustfmt.toml` and I'm using now `nightly` on fmt CI step, let me know if you think it could be an issue.